config/build.config.toml

```diff
@@ -29,8 +29,8 @@ auditor_public_keys = [
     { path = "config/keys/auditors/albus_dumbledore/albus_dumbledore.public.pem"},
 ]
 
-# RPC max message size
-rpc_max_message_size = 409600
+# gRPC max message size
+grpc_max_message_size = 4194304
```

Comment:
It seems not necessary to change the name. The old name is used in many places. It is better to keep them there.